### DISCUSSION TITLE
Windows 10 opens the software with a white screen and no content (monitor/display turned off)

### DISCUSSION THREAD

**Discussion by wangmingdegithub** [rustdesk/rustdesk#10344] [General]
![image](https://github.com/user-attachments/assets/9c801d57-c187-4ff6-9b42-7f217493d1a9)

**Comment by open-trade** [comment]
https://github.com/rustdesk/rustdesk/issues/6682
https://github.com/rustdesk/rustdesk/issues/4296#issuecomment-2183830202
https://github.com/rustdesk/rustdesk/issues/6139#issuecomment-1805681524
https://github.com/rustdesk/rustdesk/issues/418#issuecomment-1233024266

**Reply by wangmingdegithub** [reply]
Yes, my remote display is not turned on

**Reply by wangmingdegithub** [reply]
Good, I installed a virtual monitor to solve the problem
https://github.com/rustdesk/rustdesk/issues/6682#issuecomment-1874791461
